text/trace/0220-messaging-semantic-conventions-span-structure.md

```diff
@@ -0,0 +1,667 @@
+# Span structure for messaging scenarios
+
+This OTEP aims at defining consistent conventions about what spans to create
+for messaging scenarios, and at defining how those spans relate to each other.
+
+This OTEP is based on [OTEP 0173](0173-messaging-semantic-conventions.md),
+which defines basic terms and describes messaging scenarios that should be
+supported by messaging semantic conventions. It also relies on context
+propagation requirements put forth in the existing [semantic conventions](https://github.com/open-telemetry/semantic-conventions/blob/main/specification/trace/semantic_conventions/messaging.md#context-propagation)
+and detailed in [OTEP 0205](0205-messaging-semantic-conventions-context-propagation).
+
+* [Terminology](#terminology)
+* [Motivation](#motivation)
+* [Stages of producing and consuming messages](#stages-of-producing-and-consuming-messages)
+* [Trace structure](#trace-structure)
+* [Proposed changes and additions to the messaging semantic conventions](proposed-changes-and-additions-to-the-messaging-semantic-conventions)
```

Comment:
Are you missing all the "#"s from here on, or does GH make this work?

Reply:
Fixed. It seems GH made it work, however, I added the hashes nevertheless.